tensorflow-ops/src/TensorFlow/Gradient.hs

```diff
@@ -667,6 +667,18 @@ opGrad "SparseSegmentSum" _ [toT -> x, toT -> y, toT -> t] [dz] =
 opGrad "LabelClasses" _ _ _ = [Nothing, Nothing]
 opGrad "LabelWeights" _ _ _ = [Nothing]
 opGrad "Size" _ _ _ = [Nothing]
+opGrad "Tile" _ [toT -> x, toT -> multiples] [dz] =
+    [Just inputGrad, Nothing]
+  where
+    -- shapeFixedInputGrad = CoreOps.reshape inputGrad inputShape -- TODO (jcberentsen) This may be unneccessary
```

Comment:
The `set_shape` call in the python version is for static shape inference, which we don't have any support for yet. Maybe worth including a TODO for.